packages/main/test/samples/TimePicker.sample.html

```diff
@@ -0,0 +1,39 @@
+<header>
+	<h2 class="control-header">TimePicker</h2>
+</header>
+
```

Comment:
you have to show the @ since tag, refer to a sample with such

Reply:
Done